src/compas/scene/sceneobject.py

```diff
@@ -101,7 +101,7 @@ def __init__(
         **kwargs  # type: dict
     ):  # fmt: skip
         # type: (...) -> None
-        if not isinstance(item, Data):
+        if not isinstance(item, (Data, type(None))):
```

Comment:
should this not be: `if item and not isinstance(item, Data)`?

Reply:
Indeed, looks more elegant, updated!